index.html

```diff
@@ -50,5 +50,162 @@ <h1 class="head-h1">Hey there.<br>I'm Edu.<span><h2>I'm a software developer</h2
+          <div class="project-text">
+            <h2>Profesional Art<br>Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul>
+              <li class="list-area">HTML</li>
+              <li class="list-area">Bootstrap</li>
+              <li class="list-area">Ruby</li>
+            </ul>
+          </div>
+          <a tabindex="0" class="btn-action large-btn-area btn" href="#">See Project</a>
+        </div>
+      </div>
+      <div class="project-box" id="project-4">
+        <div class="project-content">
+          <div class="project-text">
+            <h2>Profesional Art<br>Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul>
+              <li class="list-area">HTML</li>
+              <li class="list-area">Bootstrap</li>
+              <li class="list-area">Ruby</li>
+            </ul>
+          </div>
+          <a tabindex="0" class="btn-action large-btn-area btn" href="#">See Project</a>
+        </div>
+      </div>
+      <div class="project-box" id="project-5">
+        <div class="project-content">
+          <div class="project-text">
+            <h2>Profesional Art<br>Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul>
+              <li class="list-area">HTML</li>
+              <li class="list-area">Bootstrap</li>
+              <li class="list-area">Ruby</li>
+            </ul>
+          </div>
+          <a tabindex="0" class="btn-action large-btn-area btn" href="#">See Project</a>
+        </div>
+      </div>
+      <div class="project-box" id="project-6">
+        <div class="project-content">
+          <div class="project-text">
+            <h2>Profesional Art<br>Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul>
+              <li class="list-area">HTML</li>
+              <li class="list-area">Bootstrap</li>
+              <li class="list-area">Ruby</li>
+            </ul>
+          </div>
+          <a tabindex="0" class="btn-action large-btn-area btn" href="#">See Project</a>
+        </div>
+      </div>
+    </section>
+    <section class="about_me" id="about">
+      <div class="about_me-container">
+        <div class="about_me-text">
+          <h2>About me</h2>
+          <p>Hello I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          <a tabindex="0" class="btn-action btn btn-area" href="#">Get my resume</a>
+        </div>
```

Comment:
- [OPTIONAL] Please consider adding a bit of gap between the elements of this `div` ⛏🤓 You can use `gap: 2rem or your preffered value` for `.about_me-text`. See the images below for clarity 💡

Your design | Given design
---| ---|
![image](https://github.com/Xecyc/My-Portfolio-Project/assets/106119105/f561217a-cd3e-4634-afc7-fbd54a78abe0) | ![image](https://github.com/Xecyc/My-Portfolio-Project/assets/106119105/191eb05c-13f6-47eb-ba38-2c441a78a3df)